modules/fabric-project/variables.tf

```diff
@@ -0,0 +1,110 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "parent_id" {
+  description = "Id of the resource under which the folder will be placed."
+}
+
+variable "parent_type" {
+  description = "Type of the parent reosurce, defaults to organization."
+  default     = "organization"
+}
+
+variable "prefix" {
+  description = "Prefix used to generate project id and name"
+}
+
+variable "name" {
+  description = "Project name and id suffix."
+}
+
+variable "billing_account_id" {
```

Comment:
Let's standardize on the same name for shared variables with the standard module:
- billing_account
- activate_apis

Reply:
I'm not too keen on the `activate_apis` name, as it masks the actual resource name. But can see the benefits of standardizing, lemme fix that.